supportedBackends/google/pipelines/batch/src/main/scala/cromwell/backend/google/pipelines/batch/GcpBatchInitializationActor.scala

```diff
@@ -110,6 +120,82 @@ class GcpBatchInitializationActor(batchParams: GcpBatchInitializationActorParams
     } yield encrypted
   }
 
+  private def vpcNetworkAndSubnetworkProjectLabelsFuture(): Future[Option[VpcAndSubnetworkProjectLabelValues]] = {
+    def googleProject(descriptor: BackendWorkflowDescriptor): String = {
+      descriptor.workflowOptions.getOrElse(WorkflowOptionKeys.GoogleProject, batchParams.batchConfiguration.batchAttributes.project)
+    }
+
+    def projectMetadataRequest(vpcConfig: VirtualPrivateCloudLabels): Future[HttpRequest] = {
+      Future {
+        val credentials = vpcConfig.auth.credentials(workflowOptions.get(_).get, List(CloudLifeSciencesScopes.CLOUD_PLATFORM))
```

Comment:
`workflowOptions.get(_).get` seems to be `Option#get` which is discouraged due to the lack of a proper error message, an alternative for this case could be `workflowOptions.get(_).getOrElse(throw new RuntimeException("Unable to find the necessary workflow option))`, ideally, add more context to the error message

Reply:
Thanks!  Adjusted.